frontend/packages/kubevirt-plugin/src/components/form/size-unit-form-row.tsx

```diff
@@ -33,41 +34,47 @@ export const SizeUnitFormRow: React.FC<SizeUnitFormRowProps> = ({
   isDisabled,
   onSizeChanged = () => undefined,
   onUnitChanged = () => undefined,
-}) => (
-  <FormRow
-    key="size"
-    title={title}
-    fieldId={prefixedID(id, 'size')}
-    isRequired={isRequired}
-    validation={validation}
-  >
-    <Split>
-      <SplitItem isFilled>
-        <Integer
-          isFullWidth
-          isValid={!isValidationError(validation)}
-          isDisabled={isDisabled}
-          id={prefixedID(id, 'size')}
-          value={size}
-          isPositive
-          onChange={React.useCallback((v) => onSizeChanged(v), [onSizeChanged])}
-          aria-label={`${title} size`}
-        />
-      </SplitItem>
-      <SplitItem>
-        <FormSelect
-          className="kubevirt-size-unit-form-row__unit"
-          onChange={React.useCallback((u) => onUnitChanged(u as BinaryUnit), [onUnitChanged])}
-          value={unit}
-          id={prefixedID(id, 'unit')}
-          isDisabled={isDisabled}
-          aria-label={`${title} unit`}
-        >
-          {(units || getStringEnumValues<BinaryUnit>(BinaryUnit)).map((u) => (
-            <FormSelectOption key={u} value={u} label={toIECUnit(u)} />
-          ))}
-        </FormSelect>
-      </SplitItem>
-    </Split>
-  </FormRow>
-);
+}) => {
+  const { t } = useTranslation();
+
+  const titleText = title || t('kubevirt-plugin~Size');
```

Comment:
@pcbailey @rawagner this breaks the wizard

![wizard](https://user-images.githubusercontent.com/3648838/100481794-9cc1cf00-30f5-11eb-9aa9-2960d57a4ccf.png)